tests/sentry/issues/auto_source_code_config/test_task.py

```diff
@@ -164,52 +190,13 @@ def setUp(self) -> None:
         self.platform = "javascript"
         self.event = self.create_event(
             [
-                {
-                    "filename": "../node_modules/@sentry/browser/node_modules/@sentry/core/esm/hub.js",
-                    "in_app": False,
-                },
-                {
-                    "filename": "./app/utils/handleXhrErrorResponse.tsx",
-                    "in_app": True,
-                },
-                {
-                    "filename": "some/path/Test.tsx",
-                    "in_app": True,
-                },
-                {
-                    "filename": "sentry/test_app.tsx",
-                    "in_app": True,
-                },
+                {"filename": "../node_modules/@sentry/foo/esm/hub.js", "in_app": False},
+                {"filename": "./app/utils/handleXhrErrorResponse.tsx", "in_app": True},
+                {"filename": "some/path/Test.tsx", "in_app": True},
+                {"filename": "sentry/test_app.tsx", "in_app": True},
             ]
         )
 
-    def test_find_stacktrace_paths_single_project(self) -> None:
-        stacktrace_paths = identify_stacktrace_paths(self.event.data)
```

Comment:
Any tests testing `identify_stacktrace_paths` have been moved to `test_stacktraces.py`.